tools/rna_tools/mlocarna/macros.xml

```diff
@@ -0,0 +1,102 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="1.8.11">locarna</requirement>
+        </requirements>
+    </xml>
+    <xml name="stdio">
+        <stdio>
+            <exit_code range="1:" />
+        </stdio>
+    </xml>
+
+    <xml name="version">
+        <version_command>
+            <![CDATA[
+                     mlocarna --version
+            ]]>
+        </version_command>
+    </xml>
+    
+    <xml name="common_scoring_parameters">
+        <param name="struct_weight" label="Structure Weight" type="integer" 
+               value="200" help="--struct-weight" />
+        <param name="indel_opening" label="Indel Opening Score" type="integer"
+               value="-500" help="--indel-opening" />
+        <param name="indel" label="Indel score" type="integer" value="-350" help="--indel" />
+        <param name="tau" type="integer" value="50" 
+               label="Sequence contribution at structure match in percent"
+               help="--tau factor"/> 
+
+        <conditional name="sequence_score">
+            <param name="sequence_score_selector" type="select" label="Type of sequence score contribution">
+                <option value="ribofit">Use ribofit</option>
```

Comment:
You could use as value="--ribofit true" and save a little bit of code above in the command section, not sure it's worth the effort and maybe it makes the code not so readable.